### ISSUE TITLE
Add info for all the guests

### ISSUE THREAD

**Issue by jensimmons** [jensimmons/thewebahead#1]
I've created a "content type" in Drupal called Guest. Every guest on any show will have a "content" of this content type. AKA (in non Drupal-speak): there's a separate page/database-entry for each Guest. 

Right now I've just added one guest to the system (as a test): Ethan Marcotte. 
http://dev-webahead.gotpantheon.com/guest/ethan-marcotte
UPDATE: oh, wait, I made two:
http://dev-webahead.gotpantheon.com/guest/peter-lubbers

For this mission, if you decide to accept, to create a new page for every guest, filling out the content form for each one with info you scrounge up on teh internets.

You make a new Guest page by clicking on Create Content, and following the flow. 

![welcome to the web ahead the web ahead 2014-08-05 19-40-07 2014-08-05 19-41-27](https://cloud.githubusercontent.com/assets/108474/3820547/14a688a6-1cfa-11e4-88de-507d493bd4d2.jpg)

It should be easy to figure out, but definitely comment below in this thread if it's not easy. Drupal has a zillion options for how to set-up Drupal, and I want to set-up things so it's easy. Your feedback will help me make adjustments.

Once you get in there, the form looks like this:

![edit guest peter lubbers the web ahead 2014-08-05 19-39-28 2014-08-05 19-39-37](https://cloud.githubusercontent.com/assets/108474/3820557/3b024e22-1cfa-11e4-94d2-eda136fa8732.jpg)

The goal is to get the correct name, a great bio, an awesome photo, a set of cool main links for each person, and to link each guest to the shows they were on. Remember, some guests have been on 2-3 shows. 

Once all of the guests are in the system, I'll set things up so that each guest's info shows up on each show page. And then I can make a grid of guests, and all kinds of other awesome things. 

Yay!

Have some fun figuring out what the bio and photo should be. It's up to you to decide what you think will work best on the site/ :D

**Comment by jensimmons** [comment]
I just added a handy page that Lists the Guests who are already in the system, so that it's easier to see who's been done already:

![list of guests the web ahead 2014-08-05 19-48-12 2014-08-05 19-48-19](https://cloud.githubusercontent.com/assets/108474/3820598/12c93866-1cfb-11e4-8165-03888e4c1fe8.jpg)

BTW, in case it's not obvious, this is NOT HOW THE WEBSITE IS GOING TO LOOK! This theme is just the default Drupal theme. I always start out building a Drupal site by focusing on the content, the system, the functionality, etc. I'll start the theme later. 

Also, this main menu of tabs has nothing to do with what the actual website will have later. I just keep throwing handy pages into the system, and putting links in the main menu to make it easier to do all these content tasks. Those things will disappear when they aren't needed anymore. 

Thanks!

**Comment by jensimmons** [comment]
I'm noticing that the photos that have been loaded so far are kind of visually all over the place. A bit messy. In part this is because the coloring of everything is so varied (different color temperatures, etc), but it's also because everyone's head is such a different size. 

![list of guests the web ahead 2014-08-07 18-10-37 2014-08-07 18-10-39](https://cloud.githubusercontent.com/assets/108474/3849984/b1c77f4c-1e7f-11e4-91d3-329396eeed79.jpg)

See how HUGE Karen's face is? And Manu? While Peter's face is tiny...

Let's see if we can get everyone's photo to be about the same kind of shot. I really like Tobias' photo. Derek, Sara, and John Albin are about the same size. If you imagined just those four shots, you'd see how more even that is. Like this: 

![list of guests the web ahead 2014-08-07 18-13-04 2014-08-07 18-13-06](https://cloud.githubusercontent.com/assets/108474/3850009/0825c52e-1e80-11e4-8cc1-877577a8fe4a.jpg)

Much better, right? So let's try to do that. 

AND all images should be cropped into a SQUARE before uploading, I'm realizing. Drupal will cut them, but if we don't crop them a head of time, Drupal robots won't do as good of a job as a human will. It'd be great to get everyone's eyes at about the same height, as well as the same sized head.

These won't be perfect, but we can get them more consistent that what I was doing the other day (see my handiwork above).

**Comment by jensimmons** [comment]
Man this one is tiring! I've finished all the guests for episode 71+ (including upcoming shows). I see you in there helping @JasonNeel, thanks. I hope to do at least a few more tonight. I'll keep going from the most-recent back in time.

**Comment by JasonNeel** [comment]
Oh I hope I wasn't stepping on your toes there! I figured I could do the write-ups for the guests of the shows I transcribed at least.

**Comment by jensimmons** [comment]
Ok, I'm up to ep66. I replaced some photos that were entered before. And made everything into circles. That looks better.

Twenty-eight guests done, only forty-eight more to go!

**Comment by jensimmons** [comment]
Oh, you weren't stepping on my toes. Step away if you'd like! I figure you, or others, can start with ep4 and work forward in time. Or jump into the middle in any random place. I keep checking to see if others have entered any, so I won't duplicate. And that's why I'm reporting on where I'm at — so others can stay away from that general vicinity.... \o/

**Comment by jensimmons** [comment]
Ok, I just finished another 7 — through episode 60.

**Comment by JasonNeel** [comment]
You're a machine!

**Comment by jensimmons** [comment]
I just want these done! And in the system, so I can design using them. Just did another few, to bring the set to half-way:

![list of guests the web ahead 2014-08-12 16-24-42 2014-08-12 16-24-45](https://cloud.githubusercontent.com/assets/108474/3897091/c5c60800-225e-11e4-9aee-0ab6a4aa8564.jpg)

That's _half_. 
Wow.

(54 is the next show in the Going-Backwards-Through-Shows Method.)

**Comment by JasonNeel** [comment]
Jonathan Stark (episdode 7) and Pat Aufderheide (episode 4) are done as well.

**Comment by jensimmons** [comment]
Sweet. Pat Aufderheide is great, isn't she? I'm working on the theme now... so prepare yourself for a visual JUMPTOSOMEWHEREELSE. And don't worry about me stepping on guests. The Drupallyness is getting to me. Time for some custom something to get started...

**Comment by JasonNeel** [comment]
She certainly seems like it. Might have to go back and listen to her episode. 

And great! I'm excited to see what's coming!

**Comment by jensimmons** [comment]
Ok, ep44 is the next one to do. 

(I applied the theme, but things exploded. It needs more work before I can put it on dev.)

**Comment by jensimmons** [comment]
ep31 is next. 

I'm really stopping for now. Help from others now would be great. Just 19ish people left.

**Comment by mattsugihara** [comment]
Starting at ep28 (Samantha Warren) and working down (towards 0)

**Comment by jensimmons** [comment]
Between you, Matt, and me, we are at ep 25 being next. Oh, plus a lot of the early episodes are done... so we just have episode 10-25 left. Twelve people!

**Comment by jensimmons** [comment]
I just saw that @JasonNeel added 6 more. ONLY SIX LEFT! I'm going to see if I can bang these out. I'd love to have all the guests in the system :smile:

**Comment by JasonNeel** [comment]
Heck yeah! From what I can see PPK, Mat Marquis, LukeW, Horace Dediu, Lyza Gardner, and Alex Barth are all that’s left.

**Comment by jensimmons** [comment]
YAYA! Wanna do them? I was going to, but I'm too excited about changing DNS records. I've got everything in place to setup thewebahead.net. Soon, dev.thewebahead.net will replace the long dev URL (http://dev-webahead.gotpantheon.com). So lemme do that. And if those 6 are still waiting, I'll do them later.

**Comment by JasonNeel** [comment]
Yeah, I’ll try and tackle them!

**Comment by JasonNeel** [comment]
Wilto, Lyza, and LukeW done. Horace, Alex, and PPK left.

**Comment by JasonNeel** [comment]
Double-check, but I think I got the last few remaining added to the site this evening.

**Comment by jensimmons** [comment]
WOOOHOOOOO!!!!!!

Gosh this page looks amazing: http://dev.thewebahead.net/guests. (Content-wise, not visual styling. That's still coming later.)

Thank you to everyone who helped do this — especially @JasonNeel who added 18 people. 

I'm sure there are details to improve and mistakes to fix (like I added Eric Meyer the other day without linking him to any of the 11 episodes he's been on — wha? Fixed yesterday.) But overall GREAT JOB EVERYBODY!! THIS IS DONE! Task #1, complete.
